test/browser/webgl_create_context_swapcontrol.c

```diff
@@ -23,5 +23,12 @@ int main() {
   EMSCRIPTEN_RESULT res = emscripten_webgl_make_context_current(context);
   assert(res == EMSCRIPTEN_RESULT_SUCCESS);
   assert(emscripten_webgl_get_current_context() == context);
+  emscripten_webgl_destroy_context(context);
+  // Test that creating a context for a canvas that's already been used once succeeds
+  context = emscripten_webgl_create_context("#canvas", &attrs);
+  assert(context > 0); // Must have received a valid context.
+  res = emscripten_webgl_make_context_current(context);
```

Comment:
Which variant of this test fails without the fix?   Is there some kind of thread involved here or are we just rendering to and offscreen buffer on the main thread?

Reply:
This error shows up even without threads, so I thought using the existing webgl explicit swap control test would be simplest.

Reply:
Presumably the issue only shows up when `USE_OFFSCREEN_FRAMEBUFFE` is used above?

Reply:
It's true with and without offscreen framebuffer, it really is triggered by the transfer of the offscreen canvas.

Reply:
I see, but there in this test is offscreencanvas used then?

Reply:
It's used because OFFSCREENCANVAS_SUPPORT is 1 in the build settings and explicitSwapControl is true in the context.